content/docs/configuration/content.md

```diff
@@ -6,58 +6,214 @@ title = "Configuration"
 
 [sidebar]
   enable = true
+  sticky = true
```

Comment:
Sticky navbar isn't scrollable. @mpourismaiel

Reply:
Also the indentation isn't working correctly. Levels are intended more than they should be.

Reply:
Added an issue for sticky bug in Syna.